server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,66 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// 1 - Create a new team event type
+router.post("/api/team-event", auth, (req, res) => {
+  if (req.body.duration) {
+    //
+    db.TeamEventType.create({ ...req.body, teamId: req.teamId })
```

Comment:
You should be able to keep this as EventType depending on Nellie's approach

Reply:
I think it could make sense to keep it as EventType, then change the EventType schema to include a teamMembers array or a teamId

Reply:
Thanks Dan and Nellie! I will adjust the routes to your approach.